apps/meteor/tests/e2e/omnichannel/omnichannel-triggers-setDepartment.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { IS_EE } from '../config/constants';
+import { createAuxContext } from '../fixtures/createAuxContext';
+import { Users } from '../fixtures/userStates';
+import { HomeOmnichannel, OmnichannelLiveChatEmbedded } from '../page-objects';
+import { createAgent } from '../utils/omnichannel/agents';
+import { addAgentToDepartment, createDepartment } from '../utils/omnichannel/departments';
+import { test, expect } from '../utils/test';
+
+declare const window: Window & {
+	RocketChat: { livechat: { setDepartment: (dep: string) => void; maximizeWidget: () => void } };
+};
+
+test.use({ storageState: Users.admin.state });
+test.describe('OC - Livechat Triggers - SetDepartment', () => {
+	test.skip(!IS_EE, 'Enterprise Only');
+	let poLiveChat: OmnichannelLiveChatEmbedded;
+	let departments: Awaited<ReturnType<typeof createDepartment>>[];
+	let departmentA: Awaited<ReturnType<typeof createDepartment>>['data'];
+	let departmentB: Awaited<ReturnType<typeof createDepartment>>['data'];
+	let agents: Awaited<ReturnType<typeof createDepartment>>[];
+	let agent1: Awaited<ReturnType<typeof createAgent>>['data'];
+	let agent2: Awaited<ReturnType<typeof createAgent>>['data'];
+	let poHomeOmnichannelAgent1: HomeOmnichannel;
+	let poHomeOmnichannelAgent2: HomeOmnichannel;
+
+	test.beforeAll(async ({ api }) => {
```

Comment:
In the steps to reproduce, it says: "Deactivate the setting `Livechat_registration_form`", should we do that here?

Reply:
Added another case with the setting off